langserver/langserver_test.go

```diff
@@ -425,6 +424,67 @@ func yza() {}
 			},
 			wantWorkspaceReferences: []string{},
 		},
+		"go hover docs": {
+			rootPath: "file:///src/test/pkg",
+			fs: map[string]string{
+				"a.go": `// Copyright 2015 someone.
+// Copyrights often span multiple lines.
+
+// Some additional non-package docs.
+
+// Package p is a package with lots of great things.
+package p
+
+import "github.com/a/pkg2"
+
+// logit is pkg2.X
+var logit = pkg2.X
+
+// T is a struct.
+type T struct {
+	// F is a string field.
+	F string
+
+	// H is a header.
+	H pkg2.Header
+}
+
+// Foo is the best string.
+var Foo string
+`,
+				"vendor/github.com/a/pkg2/x.go": `// Package pkg2 shows dependencies.
+//
+// How to
+//
+// 	Example Code!
+//
+package pkg2
+
+// X does the unknown.
+func X() {
+	panic("zomg")
+}
+
+// Header is like an HTTP header, only better.
+type Header struct {
+	// F is a string, too.
+	F string
+}
+`,
+			},
+			wantHover: map[string]string{
+				"a.go:7:9": "package p; Package p is a package with lots of great things. \n\n",
+				//"a.go:9:9": "", TODO: handle hovering on import statements (ast.BasicLit)
```

Comment:
Is this a TODO for another PR, or forgotten about (since I have no way of knowing other than to ask)?